string list_from_ocr
---
int8 flag